llvm-obs/dod-repo-conf/artful.conf

```diff
@@ -0,0 +1,249 @@
+
+Repotype: debian
+
+Preinstall: bash perl-base libperl5.26 perl-modules-5.26 sed grep coreutils debianutils
```

Comment:
I don't think it is what we want.
It won't scale as it will break when perl or other packages will be renamed.

Reply:
We were using these configurations directly from openSUSE obs reference server.
for eg https://build.opensuse.org/project/prjconf/Debian:9.0

Do we need to generate this by our own?